### PR TITLE
[FEATURE] Add browser analytics

### PR DESCRIPTION
**Pull request by gantunesr**
# Description

Track the following browser events.

- [x] Browser opened
- [x] Select search option
- [x] Open new tab
- [x] Switch network
- [x] Use browser navigation (`Home`, `Forward`, `Back`)
- [x] Share website
- [x] Reload website
- [x] Add website to favorites

### CODE REVIEW

File: app/util/analyticsV2.js [hunk 1]
```diff
@@ -47,6 +47,16 @@ export const ANALYTICS_EVENTS_V2 = {
...
 	ONRAMP_PURCHASE_CANCELLED: generateOpt('On-ramp Purchase Cancelled'),
 	ONRAMP_PURCHASE_COMPLETED: generateOpt('On-ramp Purchase Completed'),
+	// BROWSER
+	BROWSER_OPENED: generateOpt('Browser opened'),
+	BROWSER_SEARCH_USED: generateOpt('Browser search used'),
+	BROWSER_NEW_TAB: generateOpt('Browser new tab opened'),
+	BROWSER_SWITCH_NETWORK: generateOpt('Browser switch network'),
+	BROWSER_SWITCH_ACCOUNT: generateOpt('Browser switch account'),
+	BROWSER_NAVIGATION: generateOpt('Browser navigation used'),
+	BROWSER_SHARE_SITE: generateOpt('browser share site'),
```
Comment (sethkfman) [inline app/util/analyticsV2.js:57]: Upper case the 'b' in browser

File: app/components/UI/BrowserBottomBar/index.js [hunk 2]
```diff
@@ -92,22 +93,55 @@ export default class BrowserBottomBar extends PureComponent {
...
 			this.props;
 
+		const search = () => {
+			showUrlModal();
+			this.trackSearchEvent();
+		};
+
+		const back = () => {
+			goBack();
+			this.trackNavigationEvent('Go Back');
```
Comment (sethkfman) [inline app/components/UI/BrowserBottomBar/index.js:120]: Consider adding these event strings to `ANALYTICS_EVENTS_V2` constants.

**Review by sethkfman — COMMENTED** [review]
A couple of minor comments

File: app/components/UI/BrowserBottomBar/index.js (hunk 2 above)
Comment (gantunesr) [reply app/components/UI/BrowserBottomBar/index.js:120]: The string isn't an event, it's a navigation option I pass to the event `BROWSER_NAVIGATION`. Do you think I should separate them into different events?

```
trackNavigationEvent = (navigationOption) => {
	AnalyticsV2.trackEvent(AnalyticsV2.ANALYTICS_EVENTS.BROWSER_NAVIGATION, {
		option: navigationOption,
		os: Platform.OS,
	});
};
```

Comment (sethkfman) [reply app/components/UI/BrowserBottomBar/index.js:120]: Okay, I think it is fine then. Thx for the clarification.

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**